### PR TITLE
LPS-110054 Fix tests

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by leoadb** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java [hunk 1]
```diff
@@ -194,32 +194,44 @@ protected DDMFormFieldRenderingContext createDDDMFormFieldRenderingContext(
...
 		}
+		else {
+			for (DDMFormFieldValue ddmFormFieldValue : ddmFormFieldValues) {
+				Map<String, Object> changedProperties = getChangedProperties(
+					ddmFormFieldValue);
+
+				DDMFormField ddmFormField = _ddmFormFieldsMap.get(
+					ddmFormFieldValue.getName());
+
+				if (!_ddmFormRenderingContext.isReturnFullContext() &&
```
Comment (leoadb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java:221]: @marcelabc can you please double check this condition. Is this right?

**Comment by leoadb** [comment]
sent to https://github.com/marcelabc/liferay-portal/pull/224

**Pull request closed without merging**